mParticle-Appsee.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+    s.name             = "mParticle-Appsee"
+    s.version          = "2.4.1"
```

Comment:
This version will ultimately change to make our current SDK version, 7.5.x

Reply:
In such case, how will we be able to version our kit?